### PR TITLE
add network to info command

### PR DESCRIPTION
**Pull request by asifshiraz**
Fixes fedimint/fedimint#1258

### CODE REVIEW

File: client/cli/src/main.rs [hunk 1]
```diff
@@ -509,12 +510,20 @@ async fn handle_command(
             }
         }
         Command::Info => {
+            let wallet = client.wallet_client();
+            let network: String = match wallet.config.network {
+                bitcoin::Network::Bitcoin => String::from("Bitcoin"),
+                bitcoin::Network::Testnet => String::from("Testnet"),
+                bitcoin::Network::Signet => String::from("Signet"),
+                bitcoin::Network::Regtest => String::from("Regtest"),
+            };
```
Comment (elsirion) [inline client/cli/src/main.rs:519]: [`Network` implements `Serialize`](https://docs.rs/bitcoin/0.29.2/bitcoin/network/constants/enum.Network.html#impl-Serialize-for-Network), so there is no reason to manually convert it to a string here.

**Review by elsirion — CHANGES_REQUESTED** [review]
Can you remove the `Merge branch 'master' into master` commit please (look into `git reset --hard` if you don't know how) and instead locally rebase on master?

File: client/cli/src/main.rs (hunk 1 above)
Comment (asifshiraz) [reply client/cli/src/main.rs]: Fixed

**Comment by dpc** [comment]
LGTM.

Please squash with `git fetch upstream && git rebase -i upstream/master` (https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d).

Please capitalize the commit message and keep it imperative mood, somewhat descriptive. (Check existing commits https://github.com/fedimint/fedimint/commits/master to get the sense of what I mean)

File: client/cli/src/main.rs [hunk 2]
```diff
@@ -72,6 +72,7 @@ enum CliOutput {
     },
 
     Info {
+        network: String,
```
Comment (elsirion) [inline client/cli/src/main.rs:75]: nit: I think this could be of type `Network`

**Review by elsirion — APPROVED** [review]

**Comment by elsirion** [comment]
Thx for the PR, will be useful for debugging! :tada:

File: client/cli/src/main.rs (hunk 2 above)
Comment (asifshiraz) [reply client/cli/src/main.rs:75]: Will piggy back this in next PR. this one's already merged.

**Pull request merged**